assets/js/base/components/cart-checkout/shipping-rates-control-package/index.tsx

```diff
@@ -7,9 +7,10 @@ import { decodeEntities } from '@wordpress/html-entities';
 import Label from '@woocommerce/base-components/label';
 import Title from '@woocommerce/base-components/title';
 import type { ReactElement } from 'react';
-import type { Rate, PackageRateOption } from '@woocommerce/type-defs/shipping';
+import type { PackageRateOption } from '@woocommerce/type-defs/shipping';
 import { Panel } from '@woocommerce/blocks-checkout';
 import { useSelectShippingRate } from '@woocommerce/base-context/hooks';
+import type { CartShippingRateItemShippingRate } from '@woocommerce/type-defs/cart';
```

Comment:
This gets quite confusing naming wise when we start throwing in "item" since item can refer to many things. Would it make sense to stick with just "package" and "rate" terminology? e.g. CartShippingPackage which contains CartShippingPackageRate[]? Or something along those lines.

Reply:
I've removed the word Item where it seemed appropriate and replaced it with `Package` and `Rate` - is this what you had in mind?